bluesky/plans.py

```diff
@@ -1272,7 +1274,7 @@ def spiral(detectors, x_motor, y_motor, x_start, y_start, x_range, y_range, dr,
                          'x_motor': repr(x_motor), 'y_motor': repr(y_motor),
                          'x_start': x_start, 'y_start': y_start,
                          'x_range': x_range, 'y_range': y_range,
-                         'dr': dr, 'nth': nth, 'tilt': tilt,
+                         'dr': dr,'dr_y',dr_y, 'nth': nth, 'tilt': tilt,
```

Comment:
Should it be `'dr_y':  'dr_y`? Don't forget about blank spaces.

Reply:
Thanks @mrakitin, I have made the changes and the Travis build is in progress. Still getting use to the required format !-).